src/main/java/logic/unzip/UnzipFileToDirectoryController.java

```diff
@@ -0,0 +1,61 @@
+package logic.unzip;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+
+public class UnzipFileToDirectoryController implements UnzipFileToDirectoryable{
+    @Override
+    public void unzip(Path source, Path target) {
+        try {
```

Comment:
za dużo logiki

Reply:
Usunęłam try z metody skoro w głównej metodzie wywołującej i tak robię obsługę błędów jest to zbyteczne tutaj.